### DISCUSSION TITLE
link open in new tab

### DISCUSSION THREAD

**Discussion by sijucj** [sqlpage/SQLPage#626] [General]
I have added some external links on the SQL page. If I click on a link, I want it to open in a new tab using the HTML target attribute target='_blank'. Let me know how to achieve this feature through an SQL query

the sample sql follows
"select 'SQLPage''s website' as contents,
    'https://sql.datapage.app/your-first-sql-website/' as link,
    true as italics;"

my requirement is to open the above link to new tab

**Comment by lovasoa** [comment]
The button component has what you need: https://sql.datapage.app/component.sql?component=button

**Reply by sijucj** [reply]
> The button component has what you need: https://sql.datapage.app/component.sql?component=button

Thank you for your quick reply, i am working on the top menu, please find out the following image, XYZ Home and ABC Home menu i want to open in a new tab, is it possible with Button 

![image](https://github.com/user-attachments/assets/c88ad1a6-8f5b-47cb-8cb8-13f882b8b15b)

**Reply by lovasoa** [reply]
Hi! It's not yet possible to open menu items in new tabs. 

Would you be interested in contributing this feature to SQLPage yourself? 

It's a one line change in https://github.com/lovasoa/SQLpage/blob/main/sqlpage/templates/shell.handlebars

**Reply by sijucj** [reply]
Hi, I have created the following pull request regarding the above feature

https://github.com/lovasoa/SQLpage/pull/632
